draft-js-mention-plugin/src/mentionSuggestionsStyles.css

```diff
@@ -16,3 +16,13 @@
   box-sizing: border-box;
   transform: scale(0);
 }
+
+.mentionSuggestionsHeading:global(.first) {
```

Comment:
Am I'm correct that `:global` would create conflict if people use multiple mention plugins with different themes.